vmb/templates/matrimony/match/response.html

```diff
@@ -0,0 +1,161 @@
+{% extends "matrimony/emails/base.html" %}
```

Comment:
Why is it extending from `email` base template? We needn't use `table` in web templates. It's only necessary for email templates.